docs/usage/configuration.md

```diff
@@ -25,22 +25,22 @@ Type notations are formatted like so:
 Parameter name | Docker variable | Description
 --- | --- | -----
 <a name="configUrl"></a>`configUrl` | `CONFIG_URL` |  `String`. URL to fetch external configuration document from.
-<a name="dom_id"></a>`dom_id` | `DOM_ID` |`String`, **REQUIRED** if `domNode` is not provided. The id of a dom element inside which SwaggerUi will put the user interface for swagger.
```

Comment:
`SwaggerUi` here refers to the `SwaggerUI` JavaScript construct (which is commonly used in this document). Should probably fix the casing here though, but not change to `Swagger UI`.

Reply:
✔️